pennylane/templates/subroutines/out_poly.py

```diff
@@ -444,3 +463,88 @@ def compute_decomposition(
         list_ops.append(qml.adjoint(qml.QFT)(wires=output_adder_mod))
 
         return list_ops
+
+
+def _out_poly_decomposition_resources(num_output_wires, num_work_wires, mod, coeffs_list) -> dict:
+    num_output_adder_mod = num_output_wires + 1 if num_work_wires else num_output_wires
+
+    resources = Counter(
+        {
+            resource_rep(qml.QFT, num_wires=num_output_adder_mod): 1,
+        }
+    )
+
+    coeffs_dic = dict(coeffs_list)
+
+    for item in coeffs_dic:
+
+        if not (1 in item):
```

Comment:
To appease code factor
```suggestion
        if 1 not in item:
```